.meta.toml

```diff
@@ -39,11 +53,13 @@ testenv-commands-pre = [
     "{envbindir}/buildout -c {toxinidir}/buildout.cfg buildout:directory={envdir} buildout:develop={toxinidir} buildout:root-directory={toxinidir}",
     ]
 testenv-commands = [
+    "# The universal2 environment only runs Zope's own tests",
+    "universal2: {envdir}/bin/test {posargs:-vc}",
+    "# All other environments run the expanded alltests script",
```

Comment:
Why?

Reply:
Because when pre-installing the additional packages in the universal2 environment the alltests script will point to the virtual environment's site-packages folder multiple times, which means all kinds of undesired tests are run (and fail). If you know a better solution I am all ears.

Reply:
I agree that we can avoid the "foreign" (= not genuine Zope) tests in the `Mac` environment: the tests are executed (even on `Mac`) as part of the checks of their respective project; proper interoperability with Zope is checked in the other (non `Mac`) environments and interoperability problems should not be environment specific.